src/main/java/racingcar/domain/CarDTO.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain;
+
+public class CarDTO {
+    private final String carName;
```

Comment:
중복인 것 같아요. name만으로 의미 전달이 되는 것 같습니다~

```suggestion
    private final String name;
```

Reply:
동의합니다. 수정했습니다!